hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/downloader/CommonDownloaderProperties.java

```diff
@@ -50,7 +50,7 @@ public class CommonDownloaderProperties {
 
     @Max(1)
     @Min(0)
-    private float consensusRatio = (float) 1 / 3;
+    private double consensusRatio = 0.33333333333d;
```

Comment:
We should be exact like before and not approximate. You should be able to use BigDecimal in properties:

```suggestion
    private BigDecimal consensusRatio = BigDecimal.valueOf(1).divide(BigDecimal.valueOf(3));
```

Reply:
`BigDecimal.ONE.divide(BigDecimal.valueOf(3))` causes an exception: 
                  `java.lang.ArithmeticException: Non-terminating decimal expansion; no exact representable decimal result.` 
                  
But I can make it a BigDecimal like so: 
```
    private MathContext consensusRatioMathContext = new MathContext(11, RoundingMode.CEILING);
    @Max(1)
    @Min(0)
    private BigDecimal consensusRatio = BigDecimal.ONE.divide(BigDecimal.valueOf(3), consensusRatioMathContext);
    ```
    
 The scale of 11 may need to be increased, I will do some testing to get the exact precision needed.